packages/next-intl/src/middleware/getAlternateLinksHeaderValue.tsx

```diff
@@ -73,6 +80,10 @@ export default function getAlternateLinksHeaderValue(
           localizePathname(url);
         }
 
+        if(config.basePath && !url.pathname.startsWith(config.basePath)) {
```

Comment:
There are btw. some small formatting issues, you can fix them via ESLint:

```tsx
pnpm eslint src test --fix
```

… or autofix them on save in case you use VSCode:

```
  "[typescript]": {
    "editor.defaultFormatter": "dbaeumer.vscode-eslint",
    "editor.formatOnSave": true
  },
```

I should really consider setting up https://github.com/wearerequired/lint-action to make it easier for contributors.

Reply:
Thanks for the how-to on setting up VsCode.
Opening VSCode from the repository root path gives an error with `Parsing error: Cannot read file '/users/robjam/code/community/next-intl/tsconfig.json'. eslint` so I use the cli to format things. Maybe we can fix this with a root tsconfig.json and extend it from each workspace?

Reply:
My understanding of monorepos is that individual folders should be opened when working inside of them. E.g. when working on the `next-intl` package, you'd open this as your workspace root. If that isn't the case, ESLint will not work correctly.

[I've added an info about this](https://github.com/amannn/next-intl/commit/d6b1e8b24cbae0768d6482ac73293c2a44f662a1) to [the contributors guide](https://github.com/amannn/next-intl/blob/main/CONTRIBUTORS.md) for now.